src/iterative_ensemble_smoother/ies.cpp

```diff
@@ -425,7 +425,7 @@ MatrixXd makeX(const MatrixXd &A, const MatrixXd &Y0, const MatrixXd &R,
        and when the forward model is non-linear.
        Section 2.4.3
     */
-    if (A.rows() > 0 && A.cols() > 0) {
+    if (projection && (A.rows() > 0 && A.cols() > 0)) {
```

Comment:
Should we even be here if A has either no columns or no rows?

Reply:
You are probably right, but I don't want to remove it in this PR.
Created an issue though:
https://github.com/equinor/iterative_ensemble_smoother/issues/44